torch/nn/quantized/dynamic/modules/rnn.py

```diff
@@ -150,6 +180,34 @@ def extra_repr(self):
             s += ', bidirectional={bidirectional}'
         return s.format(**self.__dict__)
 
+    def __repr__(self):
+        # We don't want to show `ModuleList` children, hence custom
```

Comment:
It is hard to understand how the output would look here. Can you add a comment for a simple example, showing what the repr would look like?